packages/core/src/components/Tabs/Tab/Tab.tsx

```diff
@@ -4,11 +4,12 @@ import React, { FC, forwardRef, ReactElement, useRef } from "react";
 import { noop as NOOP } from "lodash-es";
 import useMergeRef from "../../../hooks/useMergeRef";
 import { getStyle } from "../../../helpers/typesciptCssModulesHelper";
-import Icon, { IconSubComponentProps } from "../../Icon/Icon";
+import Icon from "../../Icon/Icon";
 import VibeComponentProps from "../../../types/VibeComponentProps";
 import { IconType } from "../../Icon/IconConstants";
 import { ComponentDefaultTestId, getTestId } from "../../../tests/test-ids-utils";
 import styles from "./Tab.module.scss";
+import { SubIcon } from "src/types";
```

Comment:
Same
Can you change it to a relative path? same as other imports in the file? (`../../../types`)

Reply:
Sure @YossiSaadi, allow me some time I'll raise another PR.